docs/development.md

```diff
@@ -6,6 +6,11 @@ next: docs/webhooks.md
 
 To develop a Probot app, you will first need a recent version of [Node.js](https://nodejs.org/) installed. Probot uses the `async/await` keywords, so Node.js 7.6 is the minimum required version.
 
+To install Node.js (Mac OS X)
+
+ * Install [homebrew](https://brew.sh/)
+ * `brew install node`
```

Comment:
I would recommend to install Node.js from https://nodejs.org/, not via homebrew. That will work for all operating systems.

Reply:
I agree. I thought about that while moving it, but decided to not worry about it because my main goal was to get it out of the hello world example, but I'll update that now